README.md

```diff
@@ -118,6 +118,16 @@ Sample setting:
 "checker-framework.languageserver_repo": "checker-framework-languageserver"
 ```
 
+
+## LSP-related Command Line Options
+This section provides a list of the Checker Framework command line options that are related to the features of this LSP 
+plugin. You can turn on/off an option by adding/removing the corresponding flag in 
+[`commandLineOptions`](#commandlineoptions).
```

Comment:
How about adding a forward-reference from that point in the readme?
E.g. the example could include passing this option.

Reply:
Sounds good! Forward-reference added.